### ISSUE TITLE
Circular dependency detected while autoloading constant User

### ISSUE THREAD

**Issue by juzap10** [HappyFunCorp/happy_seed#67]
I create a fresh app without errors, then i try to run rake db:migrate and gets me this error 
`Circular dependency detected while autoloading constant User` 
localizated on 
`/app/admin/user.rb:2:in <top (required)>`
`spec/factories/users.rb:12:in block (2 levels) in <top (required)>`
`spec/factories/users.rb:8:in block (2 levels) in <top (required)>`
`spec/factories/users.rb:7:in block (2 levels) in <top (required)>`

**Comment by wschenk** [comment]
Did you include the admin?  If not, try deleting app/admin/user.rb

**Comment by juzap10** [comment]
Yes i include the admin, but i try your solution and that fix the problem. But now i try to run the app and gets me this error
`RuntimeError - Circular dependency detected while autoloading constant SimpleContent:`
